test/extended/operators/olm.go

```diff
@@ -3,8 +3,12 @@ package operators
 import (
 	"fmt"
 
+	"context"
+	"github.com/google/go-github/github"
```

Comment:
@bparees I import a new package here, do you know how to just update its `vendor`? I used the `make update-deps` command before, but it updated all vendor versions. It causes many package dependency issues.

Reply:
not possible as far as i know.  

updating vendor versions of things should not break things, if it does then our deps are not properly pinned.